src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -85,6 +88,35 @@ const ActiveState = ({
                 const isFocused = item === activeEntry;
                 const refValue = isFocused ? activeFeedItemRef : undefined;
                 const itemFileVersionId = getProp(item, 'file_version.id');
+                const isCurrentVersion = currentFileVersionId === itemFileVersionId;
+
+                if (hasReplies && (item.type === 'comment' || item.type === 'annotation')) {
+                    return (
+                        <ActivityItem
+                            key={item.type + item.id}
+                            className="bcs-activity-thread"
+                            data-testid="activity-thread"
+                            isFocused={isFocused}
+                            ref={refValue}
+                        >
+                            <ActivityThread
+                                currentFileVersionId={currentFileVersionId}
+                                getAvatarUrl={getAvatarUrl}
+                                isCurrentVersion={isCurrentVersion}
+                                item={item}
```

Comment:
It looks like the other activity types spread `{...item}` onto their props. Should we do the same here for consistency?